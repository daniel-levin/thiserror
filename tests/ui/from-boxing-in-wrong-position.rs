use thiserror::Error;

#[derive(Debug, Error)]
#[from(boxing)]
enum E {
    #[error("...")]
    A,
}

fn main() {}
