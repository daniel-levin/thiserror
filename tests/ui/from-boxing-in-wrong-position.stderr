error: not expected here; attribute #[from(boxing)] expected on a specific field
 --> tests/ui/from-boxing-in-wrong-position.rs:4:1
  |
4 | #[from(boxing)]
  | ^^^^^^^^^^^^^^^
